styleguide/haml_locals.rb

```diff
@@ -117,6 +117,46 @@
       ]
     ]
   },
+
+  'navigation_links' => [
+    {
+      'url' => 'https =>//www.citizensadvice.org.uk/benefits',
```

Comment:
`/benefits/` - trailing slash at the end of the URLs are important.

Reply:
Will change, though this is just sample links for storybook.